internal/controller/server/handlers/msg_handler.go

```diff
@@ -3,22 +3,55 @@ package handlers
 import (
 	"log"
 	"net/http"
+	"remote/internal/controller/server/adapter"
+	"remote/internal/model"
+
+	"github.com/gorilla/websocket"
 )
 
-func HandleMessage(w http.ResponseWriter, r *http.Request) {
-	c, err := Connect(w, r)
-	if err != nil {
-		log.Fatalln(err)
-	}
+func HandleTableMessages(game model.Game) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		c, err := Connect(w, r)
```

Comment:
`Connect`? What does this function connect to? Why is it exported?

Reply:
Renamed to connectWebSocket